code/modules/lighting/lighting_object.dm

```diff
@@ -11,8 +11,6 @@
 	layer            = LIGHTING_LAYER
 	invisibility     = INVISIBILITY_LIGHTING
 
-	blend_mode       = BLEND_ADD
```

Comment:
I do not know why, but if this is set, you cannot see darkness through the vis_contents of screen objects. This should always be overridden by the darkness plane master (which is set to BLEND_MULTIPLY)  anyway, so I have no idea why this is the case.

Reply:
@MrPerson